Doc/library/enum.rst

```diff
@@ -64,6 +64,10 @@ are not normal Python classes.  See
 Module Contents
 ---------------
 
+   :class:`EnumDict`
+
+      A subclass of :class:`dict` that tracks order and enforces unique member names.
+
```

Comment:
I understand that this is used by ``EnumType``, but I don't think it should go first in the list, unless it is expected to be very widely used. I would suggest the end of the list as it is new? Also ensure that you update the `versionadded` directives at the end of the list with this new class.